EnglishFilter/sections/antiadblock.txt

```diff
@@ -8974,3 +8974,5 @@ wolframclient.net#@#.adsbygoogle
 ! admitad ad block checker
 @@||ad.admitad.com/3rd-party/advert.js
 @@||ad.admitad.com/3rd-party/*/cookie/?f=CookieChecker.remoteTest
+! https://github.com/AdguardTeam/AdguardFilters/issues/72786
+123movieslinks.live#%#//scriptlet('abort-current-inline-script', 'String.fromCharCode', 'break')
```

Comment:
1. Add the domain to the `A new/different version of "zfgloaded" ads/popups` section. There should be 2 rules: `$$script[tag-content="break;case $."][min-length="30000"][max-length="85000"]` and `#%#//scriptlet("prevent-addEventListener", "", "[$.")`.
2. Remove your rule.

Reply:
What do you mean? I can't find zfgloaded

Reply:
You should change your rule to the existing ones. They are located in `generals_extensions.txt`.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/105330166-0a184a80-5be3-11eb-8f82-7b5f86a908bb.png)

</details><br/>
If you can't do that, I can exchange it directly. Just let me know :)